### DISCUSSION TITLE
Cloudwatch Logs: The log insights query result is a timeseries struct, but can get the result on Alerting

### DISCUSSION THREAD

**Discussion by mappei** [grafana/grafana#67078] [Alerting]
### Background
I want to be alerted by the value of a field in the CloudWatch log, assuming I have the following log structure:
| Timestamp | Message |
|-|:-|
| 2023-04-14T09:18:50.486+08:00| { "timestamp": "2023-04-14 01:18:50,486+0000", "level": "INFO", "message": { "F1": "value1", "F2": "value1.1", "data": 123 }, "cold_start": true, "function_name": "xxx", "function_memory_size": "128", "function_arn": "arn:aws:lambda:ap-southeast-1:xxx:function:xx", "function_request_id": "fdd28a87-79e2-4240-8c33-64ca40e8d453", "xray_trace_id": "1-6438a9f7-dd2721fe9371f16dbf6f445a" }|
| 2023-04-14T09:18:50.486+08:00| { "timestamp": "2023-04-14 01:18:50,486+0000", "level": "INFO", "message": { "F1": "value2", "F2": "value2.2", "data": 55 }, "cold_start": true, "function_name": "xxx", "function_memory_size": "128", "function_arn": "arn:aws:lambda:ap-southeast-1:xxx:function:xx", "function_request_id": "fdd28a87-79e2-4240-8c33-64ca40e8d453", "xray_trace_id": "1-6438a9f7-dd2721fe9371f16dbf6f445a" }|
| 2023-04-14T09:18:50.486+08:00| { "timestamp": "2023-04-14 01:18:50,486+0000", "level": "INFO", "message": { "F1": "value3", "F2": "value3.3", "data": 4 }, "cold_start": true, "function_name": "xxx", "function_memory_size": "128", "function_arn": "arn:aws:lambda:ap-southeast-1:xxx:function:xx", "function_request_id": "fdd28a87-79e2-4240-8c33-64ca40e8d453", "xray_trace_id": "1-6438a9f7-dd2721fe9371f16dbf6f445a" }|

I wrote the following statement in the query of the alert rule to query the Cloudwath log:
```
fields concat(F1,":", F2) as label
| stats sum(data) as count by bin(1h), label
```
**BUT** I got an error message: `Failed to evaluate queries and expressions: failed to execute conditions: input data must be a wide series but got type long (input refid)`

Then I put this query to the dashboard with time-series type, then no error is reported and there are three data points in the chart.
1. At 2023-04-14T09:00:00, label named `value1:value1 count`, with value 123
2. At 2023-04-14T09:00:00, label named `value2:value2.2 count`, with value 55
3. At 2023-04-14T09:00:00, label named `value3:value3.3 count`, with value 4

### What happened
Ran example and custom CloudWatch Log queries and they fail with "wide series" error.
### What you expected to happen
Query returns all matching log entries and allows me to define alerts based on them.
### Environment
- Grafana version: 9.0.4
- Data source type & version: CloudWatch Logs
- Docker image: grafana/grafana-enterprise:9.0.4
- Grafana plugins: none

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). We are migrating this discussion to an issue and closing the discussion. The issue is #82720. Feel free to continue the discussion around this there. Thank you!
